src/main/java/org/sbml/spatial/segmentation/LearningKfold.java

```diff
@@ -61,19 +61,21 @@ public static void main(String[] args) {
 		try {
             
             int batchSize = 1;
-            String home = System.getProperty("user.home");
+            //String home = System.getProperty("user.home");
  
+            String directory = System.getProperty("user.dir");
+            
             String pathToImage;
             if (args.length > 0) {
                 pathToImage = args[0];
             } else {
-                pathToImage = home + File.separator + "Raw images" + File.separator + "F01_621w1_crop13.tif";
+                pathToImage = directory + File.separator + "dataset" + File.separator + "Raw images" + File.separator + "F01_621w1_crop13.tif";
```

Comment:
Maybe better to assign a new variable that stores the path to dataset since it comes up often.

Reply:
Sure, will do so.